test/unit/auth/project-config.spec.ts

```diff
@@ -490,6 +524,7 @@ describe('ProjectConfig', () => {
     it('should set readonly property multiFactorConfig', () => {
       const expectedMultiFactorConfig = {
         state: 'DISABLED',
+        factorIds: [],
```

Comment:
Is this required when state is `DISABLED`?

Reply:
yes.  I believe there are some issue with mocha concurrency.  If I try to ran the single test file, it fails.